x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/__mocks__/request_responses.ts

```diff
@@ -121,12 +124,15 @@ export const getResult = (): SignalAlertType => ({
     index: ['auditbeat-*', 'filebeat-*', 'packetbeat-*', 'winlogbeat-*'],
     falsePositives: [],
     from: 'now-6m',
-    filter: undefined,
+    filter: null,
```

Comment:
what is the difference between `filter` and `filters`

Reply:
The difference is that `filter` does not require any `kql` at all. A user can just push a regular filter they happen to have not based on any query. It's a pure filter only.

However, per our conversations, before shipping I will more than likely remove it and only keep the query + filter to keep the API from having use cases no one is asking for.